### ISSUE TITLE
[Feature Request] Chocolatey Package

### ISSUE THREAD

**Issue by mark05e** [TheGoddessInari/hamsket#50]
Wondering if someone could work on setting up the installer for the Chocolatey package manager as it would help with easier installations and updates.

Upstream version is already available - https://chocolatey.org/packages/rambox

Thank you and keep up the good work.

**Comment by TheGoddessInari** [comment]
This should probably wait until after rebranding in #32, but great suggestion.

**Comment by pauby** [comment]
I'm looking to create a Chocolatey package from this soon.

**Comment by chwba** [comment]
I already created a package but it will be on hold until after rebranding

**Comment by moghingold** [comment]
@chwba is this ready to go?

**Comment by chwba** [comment]
@moghingold I had posted [here](https://chocolatey.org/packages/rambox-os-nightly) but they didnt reply yet. Now I just created a new package and pushed it. Will post the link here once its online.

**Comment by pauby** [comment]
@moghingold Please don't push another package which is the same as [the current one](https://chocolatey.org/packages/rambox-os-nightly) as the moderators will reject it. Work with the current maintainer on their package or if you get no response use the 'Contact Site Admins' link on the package page.

**Comment by chwba** [comment]
**Push a new package:**
Done [here](https://chocolatey.org/packages/hamsket-nightly/0.1). So after reading your reply, I am now working on deprecating the old listing. Its already finished, just didnt find how I can unlist versions from a package, can you help me out?
**Version Numbering:**
The problem in the past was that many new commits which were release as nightly kept having the same version number, and afaik its not possible to repush a new version of the package without changing the version number.

I will repush the new package with the version of the maintainer 0.60 and including the dependency pointing to the old listing, and wait for further input regarding how to efficiently deal with the version numbering.

**Comment by pauby** [comment]
>  just didnt find how I can unlist versions from a package, can you help me out?

Sure. Click the 'listed status', untick the list box and click **Save**. 

![image](https://user-images.githubusercontent.com/12760779/62890803-b938cd00-bd33-11e9-9114-763af5db9256.png)

> The problem in the past was that many new commits which were release as nightly kept having the same version number, and afaik its not possible to repush a new version of the package without changing the version number.

It's not. Once a package is approved it can't be changed.

>  will repush the new package with the version of the maintainer 0.60 and including the dependency pointing to the old listing, and wait for further input regarding how to efficiently deal with the version numbering.

That might just be a typo but it's the old package (rambox-os-nightly) that takes a dependency on the new one (hamsket-nightly). This is to allow people who are upgrading rambox-os-nightly so get hamsket-nightly without any extra work.

**Comment by chwba** [comment]
Its indeed a typo, the manual says `Add a dependency on the other package (if the package is being superseded).` and by other in this context it means the new package.

**Comment by chwba** [comment]
Ok, all done. [new](https://chocolatey.org/packages/hamsket-nightly/0.1) [old](https://chocolatey.org/packages/rambox-os-nightly/0.5.19). New one needs to get rejected once so I can push with again with the correct version numbering, which corresponds with the maintainers version numbering.

**Comment by chwba** [comment]
The package is online now: https://chocolatey.org/packages/hamsket-nightly/0.60
